kerckhoff/packages/exceptions.py

```diff
@@ -6,3 +6,9 @@ class GoogleDriveNotConfiguredException(APIException):
 
     def __init__(self, package):
         self.detail = f"Google Drive is not yet configured for {package.slug}."
+
+class PublishError(APIException):
+    status_code = 400
```

Comment:
I think it's hard to say, because some of them are due to invalid AML, and others are because the server failed to do something. I think I'll add an argument to specify the error code

Reply:
ohh I see. Yeah in that case I think an argument would be good